packages/expo-contacts/ios/ContactsModule.swift

```diff
@@ -84,7 +84,7 @@ public class ContactsModule: Module, OnContactPickingResultHandler {
 
       if let identifier {
         if let foundContact = try? getContact(withId: identifier) {
-          controller = ContactsViewController.init(forNewContact: foundContact)
+          controller = ContactsViewController.init(for: foundContact)
```

Comment:
```suggestion
          controller = ContactsViewController(for: foundContact)
```
No idea why I used explicit the initializer here 😅 Let's remove it